test/testinput/bufr_ncep_prepbufr_adpsfc.yaml

```diff
@@ -24,6 +24,8 @@ observations:
             query: "*/XOB"
           latitude:
             query: "*/YOB"
+          invalidLongitude:  # make sure that path strings can't contain sequences
```

Comment:
@rmclaren is this something that does not exist and tests that it handles it gracefully?

Reply:
It points to something that exists but the query string includes the non-repeating sequence mnemonic which should not be part of the query string... The old version of the code will crash if you do this (the fixed version detects the issue and warns the user without crashing).

Reply:
oh, that is subtle but now follow though `XOB` is in the `HEADR` that latter sequence mnemonic should not be in the query

yes not sure how to express this any better in a succinct line